tst/EnergyPlus/unit/HVACStandaloneERV.unit.cc

```diff
@@ -231,7 +232,7 @@ TEST_F(EnergyPlusFixture, HVACStandAloneERV_Test2)
 
     state->dataSize->ZoneEqSizing.allocate(1);
     state->dataSize->ZoneEqSizing(state->dataSize->CurZoneEqNum).SizingMethod.allocate(25);
```

Comment:
And then sometimes it was just allocated to a different amount since it didn't need the full number of sizing types.  The efficiency makes sense, but I will still be happy to see these all go away with the change to std::array.